data/scenarios/Challenges/gopher.yaml

```diff
@@ -117,6 +118,14 @@ entities:
     description:
       - Unearthed by industrious gophers.
     properties: [known, portable]
+  - name: rolex
+    display:
+      char: 'R'
+      attr: silver
+    description:
+      - Enables robots to use the 'watch' command.
+    properties: [known, portable]
+    capabilities: [wakeself]
```

Comment:
Fun! Is the plan to eventually add this to the default `entities.yaml`?

Reply:
I could do that.

Reply:
It doesn't have to be in this PR.  We'd have to come up with a recipe for it too.